test/Dialect/Substrait/literal.mlir

```diff
@@ -34,22 +34,22 @@
 // CHECK:      substrait.plan version 0 : 42 : 1 {
 // CHECK-NEXT:   relation
 // CHECK:         %[[V0:.*]] = named_table
-// CHECK-NEXT:    %[[V1:.*]] = project %[[V0]] : tuple<si1> -> tuple<si1, !substrait.var_char<6>> {
+// CHECK-NEXT:    %[[V1:.*]] = project %[[V0]] : tuple<si1> -> tuple<si1, !substrait.fixed_binary<10>> {
```

Comment:
Maybe this is an artifact of the diff that I am looking at but this currently looks like it's going to *replace* a varchar test. That's not what we want, is it?